src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
Also, we probably don't want to retain the `logout` query param in the location bar / browser history. Thoughts on this? Have we done anything with history state yet?